Mathlib/RingTheory/PowerSeries/Inverse.lean

```diff
@@ -77,8 +77,8 @@ theorem coeff_inv_aux (n : ℕ) (a : R) (φ : R⟦X⟧) :
     · rfl
     refine ⟨?_, fun hh ↦ H.not_le ?_⟩
     · rintro ⟨⟩
-      simpa [Finsupp.single_eq_same] using le_of_lt H
-    · simpa [Finsupp.single_eq_same] using hh ()
+      simpa only [Finsupp.single_eq_same] using le_of_lt H
```

Comment:
These changes seem fine to me - but again, knowing if they have a performance impact or not would be worthwhile.
(Doing a readability-neutral change for *no* reason might not be worth it.)